src/test/java/org/openelisglobal/person/PersonServiceTest.java

```diff
@@ -39,19 +60,11 @@ public void tearDown() {
 
     @Test
     public void createPerson_shouldCreateNewPerson() throws Exception {
-        String firstName = "John";
-        String lastname = "Doe";
-
-        Person pat = createPerson(firstName, lastname);
-
-        Assert.assertEquals(0, personService.getAllPersons().size());
```

Comment:
THis method is about creating a person so the should be a person created inside here

Reply:
Also , you need to assert the number existing persons before creating a new person